<launch>

	<!-- Play rosbag -->


	<!-- Main Python Script for PointCloud2 tests -->
	<node type="python_pointcloud2.py" pkg="rgbd_numpy" name="python_pointcloud2" output="screen" />

	<!-- Kinect Driver -->
	<include file="$(find openni_launch)/launch/openni.launch" />

	<!-- Image Viewer Window -->
	<node type="image_view" pkg="image_view" name="image_view" output="screen" >
	      <remap from="image" to="/camera/depth_registered/image_filtered"/>
	</node>


</launch>